### PR TITLE
Publish homebrew formula

### PR DESCRIPTION
**Pull request by cnr**
This configures goreleaser to publish to our homebrew tap

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,12 @@
 
 The following commands will execute scripts to fetch and install the latest [GitHub Releases](https://github.com/fossas/fossa-cli/releases) on the corresponding operating system.
 
+### Install with Homebrew (MacOS or Linux):
+```bash
+brew tap fossas/tap
```
Comment (zlav) [inline README.md]: [nit] We can simplify this to `brew install fossas/tap/fossa` so that users only need one line.

File: README.md [hunk 2]
```diff
@@ -41,6 +41,12 @@
 
 The following commands will execute scripts to fetch and install the latest [GitHub Releases](https://github.com/fossas/fossa-cli/releases) on the corresponding operating system.
 
+### Install with Homebrew (MacOS or Linux):
```
Comment (zlav) [inline README.md]: TIL homebrew is on linux. Want to include a hyperlink here for Homebrew or is that overkill?

**Comment by cnr** [comment]
We could construct a formula that follows [their guidelines](https://docs.brew.sh/Formula-Cookbook), which adds some maintenance overhead: we'd have to submit a PR to homebrew-core each time we publish a version, and bottle/upload compatible binaries. [Here's `goreleaser`'s formula on homebrew-core](https://github.com/Homebrew/homebrew-core/blob/master/Formula/goreleaser.rb)

It's also not clear whether they'd accept it:
> We frown on authors submitting their own work unless it is very popular.

I think having our own tap reaches the right tradeoffs

**Comment by zlav** [comment]
@cnr yea that sounds like a nightmare especially at the rate we should be releasing. Tests finally passed!

**Pull request merged**